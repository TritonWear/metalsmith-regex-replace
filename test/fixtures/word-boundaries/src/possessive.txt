Possessive words should be no indicator of a ninja's whereabouts.