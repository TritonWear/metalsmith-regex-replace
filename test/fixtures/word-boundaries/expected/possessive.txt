Possessive words should be no indicator of a blarg's whereabouts.